.circleci/config.yml

```
@@ -274,22 +274,22 @@ jobs:
       - run:
           name: Generate SQL content
           command: |
-            mkdir /tmp/generated-sql
-            cp -r sql/ /tmp/generated-sql/sql
+            mkdir -p /tmp/workspace/generated-sql
+            cp -r sql/ /tmp/workspace/generated-sql/sql
             ./bqetl bootstrap
             # Don't depend on dry run for PRs
             PATH="venv/bin:$PATH" ./script/generate_sql \
-              --sql-dir /tmp/generated-sql/sql/ \
+              --sql-dir /tmp/workspace/generated-sql/sql/ \
               --target-project moz-fx-data-shared-prod
       - persist_to_workspace:
-          root: /tmp/generated-sql
+          root: /tmp/workspace
           paths:
-            - sql
+            - generated-sql/sql
```

```suggestion
            - generated-sql
```

I think it's slightly clearer if we specify the base directory rather than the subdir here.